Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html

```diff
@@ -4,15 +4,31 @@ <h1> Elysia Range Simulator </h1>
     <form name="simulator" #simulator="ngForm" novalidate>
       <label>Speed (km/h):</label>
       <br>
-      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)">
+
+      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)" 
+       [formControl]="speedInput">
       <br>
+        <div *ngIf="(speedInput.errors?.min || speedInput.errors?.max) && this.buttonPressed" [ngClass] ="'error'">Speed must be between 0 - 90</div>
+        <div *ngIf="speedInput.errors?.required && this.buttonPressed" [ngClass]="'error'">Speed is required</div>
+      <br>
+
       <label>Battery Percentage (%):</label>
       <br>
-      <input name="battery"type="number" placeholder="Battery" (keyup)="onKeyBattery($event)">
+
+      <input name="battery"type="number" placeholder="Battery" (keyup)="onKeyBattery($event)"
+      class="form-control" required minlength ="0" [formControl]="batteryInput">
+      <br>
+      <div *ngIf="(batteryInput.errors?.min || batteryInput.errors?.max) && this.buttonPressed" [ngClass] ="'error'">Battery must be between 0 - 100</div>
```

Comment:
For all these errors, it would be nice if they were red, instead of white.
So instead of this:
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/29678619/216782132-7b5e673d-40cd-4c43-8591-ba3db57022ae.png">

It would be this:
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/29678619/216782270-3e862448-4195-4731-a0e7-71f90479222e.png">